src/main/java/marquez/api/CreateNamespaceResponse.java

```diff
@@ -1,35 +0,0 @@
-package marquez.api;
```

Comment:
We will eventually have an <Resource>Response.java object for all of the resources we care about.

Reply:
Will address in future reviews.